.gitignore

```diff
@@ -15,3 +15,6 @@ dist/
 # Vi temporary files
 *.swp
 
+## Kubernetes
+kubernetes/
```

Comment:
There's already a .tmp/ directory. Can you use it? See build/conf.js file for location of the .tmp folder.